sdk/core/core-client-rest/src/apiVersionPolicy.ts

```diff
@@ -21,7 +21,7 @@ export function apiVersionPolicy(options: ClientOptions): PipelinePolicy {
       if (!url.searchParams.get("api-version") && options.apiVersion) {
         // append the apiVersion with client one
         url.searchParams.append("api-version", options.apiVersion);
```

Comment:
I think we can get rid of this line since we're building up the query string ourselves.